index.html

```diff
@@ -3,9 +3,535 @@
+  style="width: 597px; height: 243px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+     
+     
+
+    </div>
+  
+
+<hr>
+
+<div id ="badge">
+
+    <h2>@Badge</h2>
+<a href="#" class="notification">
+    <span>Inbox</span>
+    <span class="badge">5</span>
+  </a>
+
+  <div class="avatar-large avatar-wrapper-large green-badge">
+    <img src="/assets/Images/avatar1.jpg" class="avatar avatar-medium">
+
+  </div>
+
+  
+
+  
+  
+
+ <p><iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28232%2C236%2C238%2C1%29&t=material&wt=none&l=auto&ds=true&dsyoff=17px&dsblur=68px&wc=true&wa=true&pv=17px&ph=14px&ln=false&fl=1&fm=Hack&fs=14px&lh=108%25&si=false&es=2x&wm=false&code=%253Ca%2520href%253D%2522%2523%2522%2520class%253D%2522notification%2522%253E%250A%2520%2520%2520%2520%253Cspan%253EInbox%253C%252Fspan%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge%2522%253E5%253C%252Fspan%253E%250A%2520%2520%253C%252Fa%253E%250A%250A%2520%2520%253Cimg%2520src%253D%2522%252Fassets%252FImages%252Favatar1.jpg%2522%2520class%253D%2522avatar%2520avatar-medium%2522%253E%250A%2520%2520%253Cspan%2520class%253D%2522badge%2520round%2522%253E%253C%252Fspan%253E%250A%250A"
+    style="width: 648px; height: 243px; border:0; transform: scale(1); overflow:hidden;"
+    sandbox="allow-scripts allow-same-origin">
+  </iframe></p> 
+</div>
+
+
+
+
+<div id="button">
+    <section class="btn-rw">
+        <h2>@Button</h2>
+        <button class="w-button  default">Default</button>
+        <button class="w-button  primary">Primary</button>
+        <button class="w-button secondary">Secondary</button>
+        <button class="w-button disabled">Disabled</button>
+        <button class="w-button link">Link</button>
+     
+    
+       </section>
+
+       <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28232%2C236%2C238%2C1%29&t=material&wt=none&l=auto&ds=true&dsyoff=17px&dsblur=68px&wc=true&wa=true&pv=17px&ph=14px&ln=false&fl=1&fm=Hack&fs=14px&lh=108%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522w-button%2520%2520default%2522%253EDefault%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520%2520primary%2522%253EPrimary%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520secondary%2522%253ESecondary%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520disabled%2522%253EDisabled%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520link%2522%253ELink%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%250A"
+  style="width: 580px; height: 213px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+       <section class="icon-btn">
+        <h3>@Icon Button</h3>
+        <p>Icon buttons with Text</p>
```

Comment:
Hey @Punithkumar18 ,
It would look better if you add some padding to all the p tag use under components heading

Reply:
Hey Deepak, I didn't notice this . I will make those changes